doc/development/development.rst

```diff
@@ -192,6 +192,11 @@ Miscelleaneous Stylistic Conventions
 #. Avoid using abreviations in variable names (e.g., use :code:`recording` instead of :code:`rec`). It is especially important to avoid single letter variables.
```

Comment:
```suggestion
#. Avoid using abbreviations in variable names (e.g. use :code:`recording` instead of :code:`rec`). It is especially important to avoid single letter variables.
```
@zm711